extensions/kafka-client/deployment/src/main/java/io/quarkus/kafka/client/deployment/KafkaDevServicesBuildTimeConfig.java

```diff
@@ -33,4 +33,31 @@
     @ConfigItem(defaultValue = "vectorized/redpanda:v21.5.5")
     public String imageName;
 
+    /**
+     * Indicates if the Kafka broker managed by Quarkus Dev Services is shared.
+     * When shared, it looks for running containers using a the label-based service discovery.
+     * If a matching container is found, it uses it, and so avoid to start a second one.
```

Comment:
Too many its. "It uses it" is ... bad. I recall I suggested a vastly different text and it took me a while to write it. But the term "Dev Services for Kafka" is just too long to be used as a subject in a sentence, unless you take extra care.

Let me try again:

```java
/**
 * Indicates if the Kafka broker managed by Quarkus Dev Services is shared.
 * When shared, Quarkus looks for running containers using label-based service discovery.
 * If a matching container is found, it is used, and so a second one is not started.
 * Otherwise, Dev Services for Kafka starts a new container.
 * <p>
 * The discovery uses the {@code quarkus-dev-service-kafka} label. The value is configured
 * using the {@code service-name} property.
 * <p>
 * Container sharing is only used in dev mode.
 */
```

Reply:
Fixed.